src/app/zap-templates/zcl/data-model/chip/global-structs.xml

```diff
@@ -26,8 +27,8 @@ TODO: Make these structures global rather than defining them for each cluster.
     <cluster code="0x0150"/> <!-- Service Area Cluster -->
     <!-- TODO: add Basic Information Cluster code="0x0028" -->
 
-    <item fieldId="0" name="LocationName"    type="char_string" length="128" isNullable="false"/>
-    <item fieldId="1" name="FloorNumber"     type="int16s"                   isNullable="true"/>
-    <item fieldId="2" name="AreaType"        type="AreaTypeTag"              isNullable="true"/>
+    <item fieldId="0" name="LocationName"    type="char_string" length="128" isNullable="false" optional="false"/>
+    <item fieldId="1" name="FloorNumber"     type="int16s"                   isNullable="true"  optional="false"/>
+    <item fieldId="2" name="AreaType"        type="AreaTypeTag"              isNullable="true"  optional="false"/>
```

Comment:
i.e., whether optional or nullable

Reply:
@hicklin Is the intention to land this as is defined in 0.7 and then modify it once the discussion mentioned has found a resolution or do you prefer to keep this open until we have settled on optional or nullable?

Reply:
Yes. But this PR aims to get the SDK to match with the 0.7 ballot. The changes that will come out of the 0.7 issues will be tracked via separate SDK issues. This should make it easier to manage and review.

Reply:
Sorry. When I made my comment, only the first comment was visible. @ReneJosefsen, yes. That is the intention. We have a lot of work riding on this PR so we want to get things merged rather than maintaining PR dependencies. This will also make it easier to review.